### PR TITLE
Wg 8 generator questions framework

### PR DESCRIPTION
**Pull request by chrismcanulty**
Import muscle group data from wger public API and render in multipleselectlist component. Once muscle group is selected, highlight it and display in bubble list to user.

JIRA tickets: https://lhl-midterm-ecommerce.atlassian.net/browse/WG-8
https://lhl-midterm-ecommerce.atlassian.net/browse/WG-28

Screen Shot 2023-03-29 at 9 53 33 PM

Screen Shot 2023-03-29 at 9 53 43 PM

Screen Shot 2023-03-29 at 9 53 50 PM

### CODE REVIEW

File: src/context/User.Context.js [hunk 1]
```diff
@@ -18,14 +21,31 @@ const UserProvider = ({children}) => {
     }
   };
 
+  const fetchMuscleGroup = async () => {
+    // TODO: think about context name?
+    try {
+      const res = await axios.get('https://wger.de/api/v2/muscle/');
+      // const { data: { results }} = await axios.get('https://wger.de/api/v2/muscle/');
```
Comment (wadesun18) [inline src/context/User.Context.js]: Remove

File: src/screens/HomeScreen.tsx [hunk 2]
```diff
@@ -39,16 +54,44 @@ export default function HomeScreen({navigation}: NativeStackHeaderProps) {
     }
   };
 
+  const muscleGroupData = () => {
+    console.log('muscleGroup', muscleGroup);
```
Comment (wadesun18) [inline src/screens/HomeScreen.tsx]: Remove

File: src/screens/HomeScreen.tsx [hunk 3]
```diff
@@ -39,16 +54,44 @@ export default function HomeScreen({navigation}: NativeStackHeaderProps) {
...
+    const temp = [];
+    for (const muscle of muscleGroup as any) {
+      if (muscle.name_en) {
+        temp.push({...muscle, key: muscle.id, value: muscle.name_en});
+      }
+    }
+    return temp;
+  };
+
+  console.log('muscleGroupData', muscleGroupData());
```
Comment (wadesun18) [inline src/screens/HomeScreen.tsx]: Remove

File: src/screens/HomeScreen.tsx [hunk 4]
```diff
@@ -39,16 +54,44 @@ export default function HomeScreen({navigation}: NativeStackHeaderProps) {
...
   return (
     <View>
-      <Header>Workout Generator</Header>
-      {/* replace navigation with modal? */}
-      <Button onPress={() => navigation.navigate('MuscleGroup')}>
-        <ButtonText>Generate</ButtonText>
+      <Header>{questions[1]}</Header>
```
Comment (wadesun18) [inline src/screens/HomeScreen.tsx]: Maybe start index should be 0

File: src/screens/HomeScreen.tsx [hunk 5]
```diff
@@ -39,16 +54,44 @@ export default function HomeScreen({navigation}: NativeStackHeaderProps) {
...
+      <MultipleSelectList
+        setSelected={setSelected}
+        data={muscleGroupData()}
+        save="value"
+        boxStyles={{marginLeft: 20, marginRight: 20, borderRadius: 20}}
+        dropdownStyles={{marginLeft: 20, marginRight: 20, borderRadius: 20}}
```
Comment (wadesun18) [inline src/screens/HomeScreen.tsx:74]: Can you create a style object for this so no inline styling

**Review by wadesun18 — APPROVED** [review]

File: src/screens/HomeScreen.tsx (hunk 5 above)
Comment (chrismcanulty) [reply src/screens/HomeScreen.tsx:74]: Will complete this later in a separate style PR

**Pull request merged**